benchmarks/java/pom.xml

```diff
@@ -0,0 +1,148 @@
+
+  <dependencies>
+    <dependency>
+	    <groupId>com.google.protobuf</groupId>
+	    <artifactId>protobuf-java</artifactId>
+	    <version>3.5.0</version>
+	</dependency>
+	<dependency>
+		<groupId>com.google.caliper</groupId>
+	    <artifactId>caliper</artifactId>
+		<version>1.0-beta-2</version>
+	</dependency>
+  </dependencies>
+
+  <build>
+    <pluginManagement>
+	  <plugins>
+		<plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-assembly-plugin</artifactId>
+			<version>2.4.1</version>
+			<configuration>
+				<!-- get all project dependencies -->
+				<descriptorRefs>
+					<descriptorRef>jar-with-dependencies</descriptorRef>
+				</descriptorRefs>
+				<!-- MainClass in mainfest make a executable jar -->
+				<archive>
+				  <manifest>
+					<mainClass>com.mkyong.core.utils.App</mainClass>
+				  </manifest>
+				</archive>
+
+			</configuration>
+			<executions>
+			  <execution>
+				<id>make-assembly</id>
+                                    <!-- bind to the packaging phase -->
+				<phase>package</phase>
+				<goals>
+					<goal>single</goal>
+				</goals>
+			  </execution>
+			</executions>
+		</plugin>
+	    <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.5.1</version>
+          <configuration>
+            <source>1.8</source>
+            <target>1.8</target>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-jar-plugin</artifactId>
+          <version>2.5</version>
+          <configuration>
+            <archive>
+              <manifest>
+                <addClasspath>true</addClasspath>
+                <mainClass>com.google.protocolbuffers.ProtoBench</mainClass>
+              </manifest>
+            </archive>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-source-plugin</artifactId>
+          <version>2.4</version>
+          <executions>
+            <execution>
+              <id>attach-sources</id>
+              <goals>
+                <goal>jar-no-fork</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <version>2.3</version>
+          <executions>
+            <execution>
+              <phase>package</phase>
+              <goals>
+                <goal>shade</goal>
+              </goals>
+              <configuration>
+                <shadedArtifactAttached>true</shadedArtifactAttached>
+                <shadedClassifierName>all</shadedClassifierName>
+                <transformers>
+                  <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
```

Comment:
What is this for? Why do we need to shade anything?

Reply:
Done by deleting shade plugin